app/controllers/api/v1/screenshots_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    # Controller methods to call a service of AWS S3
+    class ScreenshotsController < ApplicationController
+      before_action :get_versions_id, :get_locales_id
+
+      def index
+        @screenshots = Screenshot.where(version_id: @version, locale_id: @locale)
+        render json: @screenshots, include: :images
+      end
+
+      def new
+        @screenshot = Screenshot.new
+      end
+
+      def create
+        @screenshot = Screenshot.new(screenshot_params)
+        @screenshot.save!
+      end
+
+      private
+      def screenshot_params
+        params.require(:screenshot).permit(:name, :locale_id, :version_id, images: [])
+      end
+
+      def get_versions_id
```

Comment:
I think we can rename it to `set_version` and`set_locale`, as we set the instances.

Reply:
@sourabh-upadhyay  Done with the changes as mentioned.([https://github.com/lingostack/lingoqa/pull/8#discussion_r375086045])